editor/filesystem_dock.cpp

```diff
@@ -3319,6 +3424,17 @@ FileSystemDock::FileSystemDock() {
 	ED_SHORTCUT("filesystem_dock/open_in_external_program", TTR("Open in External Program"));
 #endif
 
+	folder_colors = HashMap<String, Color>();
+	folder_colors["red"] = Color("#ff4545");
+	folder_colors["orange"] = Color("#ff8f45");
+	folder_colors["yellow"] = Color("#ffe345");
+	folder_colors["green"] = Color("#80ff45");
+	folder_colors["teal"] = Color("#45ffa2");
+	folder_colors["blue"] = Color("#45d7ff");
+	folder_colors["purple"] = Color("#8045ff");
+	folder_colors["pink"] = Color("#ff4596");
+	folder_colors["gray"] = Color("#9d9d9d");
```

Comment:
I suggest converting these `Color()` to their decimal form, as this will reduce binary size slightly (no more string to compile in the binary):

 ```suggestion
	folder_colors["red"] = Color(1.0, 0.271, 0.271);
	folder_colors["orange"] = Color(1.0, 0.561, 0.271);
	folder_colors["yellow"] = Color(1.0, 0.890, 0.271);
	folder_colors["green"] = Color(0.502, 1.0, 0.271);
	folder_colors["teal"] = Color(0.271, 1.0, 0.635);
	folder_colors["blue"] = Color(0.271, 0.843, 1.0);
	folder_colors["purple"] = Color(0.502, 0.271, 1.0);
	folder_colors["pink"] = Color(1.0, 0.271, 0.588);
	folder_colors["gray"] = Color(0.616, 0.616, 0.616);
```

Reply:
Gotcha, updated in 5347641!